File: src/main/java/ar/edu/unq/epers/unidad5/model/PesoPromedio.java
package ar.edu.unq.epers.unidad5.model;

import lombok.*;
import org.springframework.data.annotation.Id;
import org.springframework.data.mongodb.core.mapping.Document;

@ToString
@Setter
@Getter
@EqualsAndHashCode
@NoArgsConstructor(access = AccessLevel.PROTECTED)

@Document
public class PesoPromedio {
    @Id
    private String nombre;
    private double valor = 0;
}

File: src/main/kotlin/ar/edu/unq/epers/unidad5/model/PesoPromedio.kt
package ar.edu.unq.epers.unidad5.model

import org.springframework.data.annotation.Id
import org.springframework.data.mongodb.core.mapping.Document

@Document
class PesoPromedio {
    @Id
    var nombre: String? = null
    var valor:Double = 0.toDouble()
}